roles/StackStorm.st2/tasks/version.yml

```diff
@@ -1,7 +1,8 @@
 ---
 # Getting the current st2 version is required to understand which 'st2_services' to restart
 - name: Get installed st2 version
-  command: /opt/stackstorm/st2/bin/python -c 'import st2common; print st2common.__version__'
+  # command: /opt/stackstorm/st2/bin/python -c 'import st2common; print st2common.__version__'
```

Comment:
```suggestion
```

Commented value could be removed.

Reply:
ah sorry, that one made it through my mistake.